### PR TITLE
Plastic Padding material, buffed touring suit, completely reworked motorcycle armor.

### PR DESCRIPTION
**Pull request by Aerin-of-the-Toast**
#### Summary
Balance "Make the touring suit and motorcycle armor much better"

#### Purpose of change

A lot of pads in stuff like elbow pads or motorcycle gear have thicknesses ranging from 12mm to 22mm or more, but using the base "plastic" material with thickness that large gave wildly inflated defensive values. When I reworked the touring suit previously, I massively understated the coverage of the pads, which actually cover a fair bit. Lastly, the motorcycle armor was just... really strange.

#### Describe the solution

New material Plastic Padding was added, with defensive values of 0.75/0.5/0,5 (since it's meant to protect from impacts) so that various pads can have more realistic thicknesses without suddenly becoming better than plate armor. This plastic padding was then added to the motorcycle armor and touring suit which I've fixed up too.

For the touring suit, you'll have to forgive me for using "my dirtbiker friend" as a source once again. For his gear at least, the padding locations are: The chest and back (with the lower stomach exposed) for the torso, the shoulders, elbows, and forearms for the arms (and the shoulder pads are often large enough to cover a significant portion of the upper arm), and for the legs the hips and thighs, knees, and shins are covered. (The shins are usually actually provided by separate touring boots. For now I've rolled them into the suit instead, but in the future I'd like to actually make them part of separate touring boots once I figure out how to make the shin guards from boots not conflict with the touring suit in layers.) This generally means much higher coverage values than before for the padding, though there are significant gaps in the limbs still, so the coverage is only 80 instead of 90. I also undervalued how stuff the leather was, according to my friend, so I've increased the encumbrance by 5.

Lastly for the motorcycle armor, I used this armor as reference: https://www.amazon.com/Motorcycle-Armor-Protective-Motocross-Protection/dp/B07MW28WJH?tag=drive09e6-20&asc_source=browser&asc_refurl=https%3A%2F%2Fwww.thedrive.com%2Freviews%2F31304%2Fbest-motorcycle-armor&th=1

It is effectively similar to the hard chest/arm guards in terms of padding, except with a thinner lycra jacket underneath instead of neoprene, and with shoulder protection. As such, the motorcycle armor was changed from cotton and kevlar (Which was pretty strange, where's the padding?) to lycra and plastic padding, and the encumbrance was dropped massively to be about the same as the hard chest/arm guards as the pads are placed to not be very restricting and the lycra is easy to move in. The armor was intentionally left _not_ STURDY since the lycra can tear easily, but the "repairs_like" was changed to repairing similar to a compression top, so it's fairly easy to repair if you have replacement lycra.

There were also a few misc changes to stuff like the environmental protection and giving both the option to block with the arms while worn similar to other suits of armor.

#### Describe alternatives you've considered

Mostly just keeping the shin protection off the touring suit via sublocations and letting survivor's shins get munched until I add a touring boot (or they craft shin guards I guess).

#### Testing

Game works with the changes.

#### Additional context

The touring suit rework I did had made the touring suit nearly unusable, and this version is actually strong enough to consider using now. It's still not ideal due to the encumbrance, but now it can actually be somewhat reliably protective and will last long. The motorcycle armor as well used to be unusably bad, but now is actually really good armor early game. Slightly better than wearing a hard chest/arm guard due to shoulder protection, but without the option to replace the chest guard with a ballistic vest.

### CODE REVIEW

**Comment by descan** [comment]
Would it be realistic to add sleeveless/separated versions of those? I'm not sure cohesive the whole is. The shoulderpads on the touring suit make me thing a sleeveless variant isn't really suitable - de-sleeving them would be cutting into those pads. The motorcycle armor might be suitable for de-sleeving though, it's hard to tell where those shoulder pads end.. The fact that they're buckled on though, inclines me to think possibly yes.

And splitting a touring suit into a top half and a bottom half, similar to a union suit -> long underwear top and bottom, might be possible based on your description?

**Comment by Aerin-of-the-Toast** [comment]
> Would it be realistic to add sleeveless/separated versions of those? I'm not sure cohesive the whole is. The shoulderpads on the touring suit make me thing a sleeveless variant isn't really suitable - de-sleeving them would be cutting into those pads. The motorcycle armor might be suitable for de-sleeving though, it's hard to tell where those shoulder pads end.. The fact that they're buckled on though, inclines me to think possibly yes.
> 
> And splitting a touring suit into a top half and a bottom half, similar to a union suit -> long underwear top and bottom, might be possible based on your description?

A lot of touring suits nowadays are made to be separate jackets and pants, so they could be split, yeah. That'd be a separate PR though. As for the sleeveless motorcycle armor, that makes sense I suppose, though I'm not sure if wearing just the sleeves would work. I think you'd have problems with it bunching down your arm. I'm not sure though, but that also seems like something for another PR if applicable.

**Comment by descan** [comment]
I think if you were to wear the sleeves of a motorcycle armor, you'd need them to buckle onto a harness or something - I don't think you can really do that in CDDA at the moment, so maybe make a de-sleeving recipe for motorcycle armour require a short rope, have the sleeves connect to each other, and have them also cover the upper torso? To represent them being connected to each other. I'm not sure how you'd do that without making them worthless to wear though, as they'd interfere with whatever you were wearing instead of the motorcycle armour vest.

(The suggestion at start was more about getting the vest than about the sleeves, but you're right, all those recipes do end up with sleeves at the end that you could use and that should be accounted for.)

**Comment by bombasticSlacks** [comment]
screen shots would be appreciated

**Comment by Aerin-of-the-Toast** [comment]
> screen shots would be appreciated

image
image

**Comment by bombasticSlacks** [comment]
you can either delete the "materials" line or change plastic to your foam mat. Either option works but it should say its the right materials.

**Comment by Aerin-of-the-Toast** [comment]
> you can either delete the "materials" line or change plastic to your foam mat. Either option works but it should say its the right materials.
> 
> Otherwise these look good!

Oops, good catch. Fixed the touring suit's materials.

**Comment by Night-Pryanik** [comment]
This will conflict with #55796?

**Comment by Aerin-of-the-Toast** [comment]
> This will conflict with #55796?

Oh heck, I had forgotten about that. So funny story about that was I had somewhat helped them make that PR at first but I hadn't heard they finished it that I remember. In either case, I'm unsure if they're still around anyway to finish the PR/ provide sources like Kevin asked, alas. Also I disagree with some of the stats they used (though some of them might've been suggested by me, I can't remember, it's been several months) compared to the ones I'm using now. 

So basically I think it's more my PR here renders that one redundant.

**Comment by bombasticSlacks** [comment]
I'm gonna merge this one since it was originally your overhaul and we've been waiting 2 months on them providing sources.
